### ISSUE TITLE
[Bug] Export Icons positions are wrong

### ISSUE THREAD

**Issue by photocoder** [lencx/ChatGPT#1160]
### Non-ChatGPT bug

- [X] This issue does not occur on chat.openai.com and only occurs on this app.

### Version

Version 1.1.0 (20230803.134154)

### Bug description

After using any first prompt, input filed changed, becomes much higher and all icons set in column order.

![Screenshot 2023-12-02 at 09 36 14](https://github.com/lencx/ChatGPT/assets/29972465/f5c56243-4375-4790-99e9-e81479be970a)

### OS

MacOS Sanoma 14.0 (23A344)

**Comment by lduperval** [comment]
Yeah, I'm not sure what happened. If you look at the export.js file, I think that's where the design of the icons are done. They used to appear just above the Settings in the left sidebar. Now they seem to be a bit messed up. I don't know enough about this to make it work...

Maybe I'll ask Bard!

**Comment by lduperval** [comment]
See: https://github.com/lencx/ChatGPT/issues/1156#issuecomment-1848587532

**Comment by chadwellens** [comment]
Same problem here... makes the app useless!

**Comment by lduperval** [comment]
@chadwellens Did you try the solution I put in the comment above yours?

**Comment by chadwellens** [comment]
@lduperval I think I'll wait for the update... but well done on finding a solution!

**Comment by asheroto** [comment]
Apparently the export feature has been removed. See https://github.com/lencx/ChatGPT/issues/1142#issuecomment-1859316613

If you go to Preferences > Control Center > Scripts and sync all scripts, it'll update everything. It will remove the export functions. No idea if the author plans on fixing this or not. :-)
